src/content/protocols/kodiak-v3.md

```diff
@@ -0,0 +1,230 @@
+
+> Accessibility score: High
+
+# Technical Analysis
+
+## Contracts
+
+| Contract Name       | Address                                                                                                               |
+| ------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| UniswapV3Factory    | [0xD84CBf0B02636E7f53dB9E5e45A616E05d710990](https://berascan.com/address/0xD84CBf0B02636E7f53dB9E5e45A616E05d710990) |
+| SwapRouter          | [0xEd158C4b336A6FCb5B193A5570e3a571f6cbe690](https://berascan.com/address/0xEd158C4b336A6FCb5B193A5570e3a571f6cbe690) |
+| SwapRouter02        | [0xe301E48F77963D3F7DbD2a4796962Bd7f3867Fb4](https://berascan.com/address/0xe301E48F77963D3F7DbD2a4796962Bd7f3867Fb4) |
+| QuoterV2            | [0x644C8D6E501f7C994B74F5ceA96abe65d0BA662B](https://berascan.com/address/0x644C8D6E501f7C994B74F5ceA96abe65d0BA662B) |
+| MixedRouteQuoterV1  | [0xfa0276F06161cC2f66Aa51f3500484EdF8Fc94bB](https://berascan.com/address/0xfa0276F06161cC2f66Aa51f3500484EdF8Fc94bB) |
+| TickLens            | [0xa73C6F1FeC76D5487dC30bdB8f11d1F390394b48](https://berascan.com/address/0xa73C6F1FeC76D5487dC30bdB8f11d1F390394b48) |
+| KodiakIslandFactory | [0x5261c5A5f08818c08Ed0Eb036d9575bA1E02c1d6](https://berascan.com/address/0x5261c5A5f08818c08Ed0Eb036d9575bA1E02c1d6) |
+| IslandRouter        | [0x679a7C63FC83b6A4D9C1F931891d705483d4791F](https://berascan.com/address/0x679a7C63FC83b6A4D9C1F931891d705483d4791F) |
+| KodiakIsland        | [0xCFe9Ee61c271fBA4D190498b5A71B8CB365a3590](https://berascan.com/address/0xCFe9Ee61c271fBA4D190498b5A71B8CB365a3590) |
+| FarmFactory         | [0xAeAa563d9110f833FA3fb1FF9a35DFBa11B0c9cF](https://berascan.com/address/0xAeAa563d9110f833FA3fb1FF9a35DFBa11B0c9cF) |
+| KodiakFarm          | [0xEB81a9EEAF156d4Cfec2AF364aF36Ad65cF9f0fa](https://berascan.com/address/0xEB81a9EEAF156d4Cfec2AF364aF36Ad65cF9f0fa) |
+| PandaFactory        | [0xac335fe675699b0ce4c927bdaa572eb647ed9f02](https://berascan.com/address/0xac335fe675699b0ce4c927bdaa572eb647ed9f02) |
+
+## Permission owners
+
+| Name          | Account                                                                                                               | Type         |
+| ------------- | --------------------------------------------------------------------------------------------------------------------- | ------------ |
+| Safe Wallet 1 | [0x21802b7C3DF57e98df45f1547b0F1a72F2CD1aED](https://berascan.com/address/0x21802b7C3DF57e98df45f1547b0F1a72F2CD1aED) | Multisig 2/4 |
+| Safe Wallet 2 | [0x7D8A57bf453ecf440093fe6484c78B91b377e638](https://berascan.com/address/0x7D8A57bf453ecf440093fe6484c78B91b377e638) | Multisig 3/6 |
+| Safe Wallet 3 | [0x8A98f7A3dF97029bd8992966044a06DEe4CC5299](https://berascan.com/address/0x8A98f7A3dF97029bd8992966044a06DEe4CC5299) | Multisig 2/3 |
+
+## Permissions
```

Comment:
please scan also the new contracts added above and list their permissions

Reply:
add also a permissions in this contract `KodiakIslandWithRouter`

Reply:
@newgnart https://berascan.com/address/0xCFe9Ee61c271fBA4D190498b5A71B8CB365a3590#code I dont see the permissions from this contract in the existing report